DSharpPlus/Net/Abstractions/RestChannelPayloads.cs

```diff
@@ -15,6 +15,9 @@ internal sealed class RestChannelCreatePayload
         [JsonProperty("parent_id", NullValueHandling = NullValueHandling.Ignore)]
         public ulong? Parent { get; set; }
 
+        [JsonProperty("topic", NullValueHandling = NullValueHandling.Ignore)]
+        public string Topic { get; set; }
```

Comment:
I believe it actually should be `Optional<string>` given that `null` topic means clear topic. That change will require propagating to all callers.